packages/api/resolvers/mutations/accounts/createUser.js

```diff
@@ -12,5 +17,15 @@ export default async function createUser(root, options, context) {
     mappedOptions.password = hashPassword(mappedOptions.plainPassword);
     delete mappedOptions.plainPassword;
   }
-  return callMethod(context, 'createUser', mappedOptions);
+  const userId = await accountsPassword.createUser(mappedOptions);
```

Comment:
If context is not submitted to createUser, it will be missing in any hooks regarding the createUser event, no?

Reply:
Still the case? Also for changePassword and I guess you should pass it with your evaluateContext function

Reply:
>If context is not submitted to createUser, it will be missing in any hooks regarding the createUser event, no?

Correct. The context wouldn't be available in the hooks. 

>Still the case? Also for changePassword and I guess you should pass it with your evaluateContext function
I have a little epiphany which is why I was reluctant to implement this request for two reasons:
- The implementation works already as is without the need for context; when we need it we can. And surely, there's a very good chance that we do  but still.
- Sometimes to add the context, this'd require us to extend accountsjs methods as was the case for `createUser` https://github.com/unchainedshop/unchained/pull/230#discussion_r497419058
and shall be for `changePassword` if we'd want to pass the context.
https://github.com/accounts-js/accounts/blob/master/packages/password/src/accounts-password.ts#L408